### PR TITLE
Converter service refactorized

### PR DESCRIPTION
**Pull request by luisdavidparra**

### CODE REVIEW

**Review by DilanOjeda — APPROVED** [review]
Looks good!

**Review by stospina — COMMENTED** [review]
Great ... It´s ok

File: CONVERTER-SERVICE/src/models/converterChecker.facade.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-@converterChecker.helper.js Copyright (c) 2022 Jalasoft
+@converterChecker.model.js Copyright (c) 2022 Jalasoft
```
Comment (florpadilla) [inline CONVERTER-SERVICE/src/models/converterChecker.facade.js]: the file name is "converterChecker.model.js" or "converterChecker.facade.js" ?

File: CONVERTER-SERVICE/src/models/documentFileConverter.model.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/*
+@videoConverter.facade.js Copyright (c) 2022 Jalasoft
```
Comment (florpadilla) [inline CONVERTER-SERVICE/src/models/documentFileConverter.model.js]: check the file name. it should be "documentFileConverter.model.js"

**Review by florpadilla — COMMENTED** [review]
Check the copyright and correct the file name at: "converterChecker.facade.js" and at "documentFileConverter.model.js"

**Review by florpadilla — APPROVED** [review]
great!

**Pull request merged**